Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -105,21 +108,43 @@ export class dotNetExe {
             } else {
                 dotnet.arg(projectFile);
             }
+            if(this.isBuildCommand()) {
+                var detailId = "";
+                var projectFilePath = path.dirname(projectFile);
+                if(this.logProjectEvents) {
+                    detailId = uuidV4();
+                    var projectFileName = path.basename(projectFile);
+                    var detailName = tl.loc("Build", projectFileName);
+                    var startTime = new Date(Date.now()).toISOString();
+                    //Todo: can we use named parameters in typescipt?
+                    tl.logDetail(detailId, "", "", "Process", detailName, 0, startTime, "", 0, tl.TaskState.Initialized);
```

Comment:
what does this line do?

Reply:
This initializes the [timeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#logdetail-create-and-update-a-timeline-record-for-a-task). Timeline records work as parent-children. We need to initialize the first timeline record. Subsequent timelines are children of the existing timeline records, which already exist.